ui.apps/src/main/content/jcr_root/apps/hclecomm/clientlibs/clientlib-base/.content.xml

```diff
@@ -3,5 +3,5 @@
     jcr:primaryType="cq:ClientLibraryFolder"
     allowProxy="{Boolean}true"
     categories="[hclecomm.base]"
-    embed="[core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,hclecomm.grid,hclecomm.title,hclecomm.text,hclecomm.button,hclecomm.accordion,hclecomm.image,hclecomm.teaser,hclecomm.carousel,hclecomm.tabs,hclecomm.footerContainer,hclecomm.navigation,hclecomm.search,hclecomm.minicart,hclecomm.productdetails,hclecomm.cartProductDisplay,hclecomm.productcart,hclecomm.productfilter,hclecomm.checkoutform,hclecomm.signin,hclecomm.signup,hclecomm.checkoutcarttotal,hclecomm.confirmation]"/>
+    embed="[core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,hclecomm.grid,hclecomm.title,hclecomm.text,hclecomm.button,hclecomm.accordion,hclecomm.image,hclecomm.teaser,hclecomm.carousel,hclecomm.tabs,hclecomm.footerContainer,hclecomm.navigation,hclecomm.search,hclecomm.minicart,hclecomm.productdetails,hclecomm.cartProductDisplay,hclecomm.productcart,hclecomm.productfilter,hclecomm.checkoutform,hclecomm.signin,hclecomm.signup,hclecomm.checkoutcarttotal,hclecomm.confirmation,hclecomm.site]"/>
```

Comment:
@DeepaliAgrwl  : hclecomm.site client lib is part of page policy . why has this been included in hclecomm.base?

Reply:
@rishabh-behl  In page policy we have included only clientlib.base and clientlib.dependencies library. That is why I have embedded it with clientlib.base